File: week-02/day2/draw-pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

number = str(input('Give me a number please: '))

for e in range(0, 8):
    if e > 0 and e %2 == 1:
        print(' ', e * number)

File: week-07/day2/draw-pyramid.js
'use strict';


// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

var star = '*';
var paddingSpace = ' ';
var lineCount = 29;

for (var i = 0; i < lineCount; i++){
    console.log(paddingSpace.repeat(lineCount-(i)) + star);
    star += '**';
};
